src/components/containers/Accordion/Accordion.module.scss

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../../styling/defs.scss' as bk;
+
+@layer baklava.components {
+  .bk-accordion {
```

Comment:
When accordion item is opened or closed, the entire accordion width is changed. Is it intentional? (Seems like it is not changed in baklava1, and I feel like the same width looks better.)

https://github.com/user-attachments/assets/4bf3df72-69e9-4bc8-a589-e22e6b14b86e

Reply:
This is actually an issue with the story, not the component itself. I was working on the `LayoutDecorator` utility component for Storybook, but it doesn't quite work yet. So what's happening is that the story here doesn't have a fixed width defined and instead just shrinks to the max content of the `Accordion`.

I just pushed a workaround. Will revisit later.